core/templates/pages/exploration-editor-page/exploration-save-and-publish-buttons/exploration-save-and-publish-buttons.component.html

```diff
@@ -67,10 +67,10 @@
         </span>
       </button>
       <button type="button" class="btn btn-light uib-dropdown-toggle oppia-discard-button protractor-test-save-discard-toggle dropdown-toggle"
-              ng-disabled="!getChangeListLength()" uib-dropdown-toggle aria-label="Discard Draft">
+              ng-disabled="isOffline || !getChangeListLength()" uib-dropdown-toggle aria-label="Discard Draft">
       </button>
       <ul uib-dropdown-menu ng-style="{ width: getChangeListLength() && !isPrivate() ? '150px' : '120px' }">
-        <li title="Discard all pending changes"><a ng-click="discardChanges()" ng-class="{'oppia-disabled-link': !getChangeListLength()}" class="dropdown-item nav-item protractor-test-discard-changes">Discard Draft</a></li>
+        <li title="Discard all pending changes"><a ng-click="discardChanges()" ng-class="{'oppia-disabled-link': isOffline || !getChangeListLength()}" class="dropdown-item nav-item protractor-test-discard-changes">Discard Draft</a></li>
```

Comment:
Should we show a tooltip explaining why this button is disabled?

Reply:
Done.